app/controllers/users_controller.rb

```diff
@@ -3,6 +3,27 @@ class UsersController < ApplicationController
   before_action :retrieve_patients, only: [:add_patient, :remove_patient]
   rescue_from Mongoid::Errors::DocumentNotFound, with: -> { head :bad_request }
 
+  def create
+    raise 'Permission Denied' unless current_user.admin?
```

Comment:
Having it choke on someone not being an admin seems like an expected outcome here -- is it more appropriate to have this return unauthorized and halt? I can see an argument for it either way:

* The former is probably more the textbook pattern, which makes it more readable / maintainable / etc. 
* OTOH, raising an error like you're doing here makes it more likely to get picked up by a monitoring service that we use in production, thereby making it easier to figure out if something sketchy is going on and someone's trying to screw with this endpoint.

@malively What are your thoughts?